lockbox2/programs/liquidity_lockbox/src/lib.rs

```diff
@@ -266,7 +240,7 @@ pub mod liquidity_lockbox {
       cpi_accounts_modify_liquidity,
       signer_seeds
     );
-    whirlpool::cpi::increase_liquidity(cpi_ctx_modify_liquidity, liquidity_amount, delta_a, delta_b)?;
+    whirlpool::cpi::increase_liquidity(cpi_ctx_modify_liquidity, liquidity_amount as u128, token_max_a, token_max_b)?;
```

Comment:
what about the approval?

Reply:
Oh sorry! the approval is above for ` token_max_a` ` token_max_b` !!

Reply:
Fix for .unwrap_or_else(|| panic!("Liquidity overflow")) in another PR?

Reply:
This will be done in a next PR

Reply:
LGTM